modules/income_and_assets/app/models/income_and_assets/saved_claim.rb

```diff
@@ -1,40 +1,37 @@
 # frozen_string_literal: true
 
-require 'pension_burial/processing_office'
 require 'income_and_assets/benefits_intake/benefit_intake_job'
 
 module IncomeAndAssets
-  ##
   # IncomeAndAssets 21P-0969 Active::Record
   # @see app/model/saved_claim
-  #
   class SavedClaim < ::SavedClaim
     # Income and Assets Form ID
     FORM = IncomeAndAssets::FORM_ID
 
-    ##
     # the predefined regional office address
     #
     # @return [Array<String>] the address lines of the regional office
-    #
     def regional_office
       ['Department of Veteran Affairs',
        'Pension Intake Center',
        'P.O. Box 5365',
        'Janesville, Wisconsin 53547-5365']
     end
 
-    ##
-    # send this Income and Assets Evidence claim to the Lighthouse Benefit Intake API
-    #
-    # @see https://developer.va.gov/explore/api/benefits-intake/docs
-    #
-    # @param current_user [User] the current user submitting the form
+    # utility function to retrieve claimant email from form
     #
-    def upload_to_lighthouse(current_user = nil)
-      return unless Flipper.enabled?(:pension_income_and_assets_clarification, current_user)
+    # @return [String] the claimant email
+    def email
+      # parsed_form['email'] # TODO add email field to the form
+      'test@example.com'
+    end
 
-      IncomeAndAssets::BenefitIntakeJob.perform_async(id, current_user&.user_account_uuid)
+    # Utility function to retrieve veteran first name from form
+    #
+    # @return [String]
+    def first_name
+      parsed_form.dig('veteranFullName', 'first')
```

Comment:
Do we want this to be the name of the person who filled out the form or always the Veterans first name?